xonsh/commands_cache.py

```diff
@@ -168,8 +168,10 @@ def _update_paths_cache(self, paths: tp.Sequence[str]) -> bool:
         updated = False
         for path in paths:
             modified_time = os.stat(path).st_mtime
-            if (path not in self._paths_cache) or (
-                self._paths_cache[path].mtime != modified_time
+            if (
+                self.env.get("ENABLE_COMMANDS_CACHE")
```

Comment:
it is best to pass the default value True here to `.get()`

Reply:
Why is it best?🤔
If the value doesn't exist it should be a bug

Reply:
sometimes the variable might be None; during unit tests.